core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -416,11 +416,11 @@ public void track(Telemetry telemetry) {
             throw new IllegalArgumentException("Instrumentation key cannot be undefined.");
         }
 
-        try {
-            telemetry.sanitize();
-        } catch (Throwable t) {
-            InternalLogger.INSTANCE.error("Exception while sanitizing telemetry: '%s'",t.getMessage());
-        }
+//        try {
```

Comment:
@Dmitry-Matveev sure I purposefully kept the comments for now, once everything is fine I will remove off the comments . Sometimes its just more handy to comment instead of moving back to older commit using git :-p